db/migrations/0018_add_lti_usage.py

```diff
@@ -0,0 +1,43 @@
+'''
+Migration for MiVideo media started hourly table
+'''
+
+from yoyo import step
+
+steps = [
+    step('''
+        CREATE TABLE IF NOT EXISTS lti_placement_types (
+            id INTEGER NOT NULL AUTO_INCREMENT,
+            description VARCHAR(100) NOT NULL,
+            table_name VARCHAR(64) NOT NULL,
+            PRIMARY KEY (id)
+        ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4;
+    '''),
+    step('''
+        INSERT IGNORE INTO lti_placement_types
+        (id, description, table_name) VALUES
+        (0, 'Zoom', 'lti_zoom_placement')
+    '''),
+    step('''
+        CREATE TABLE IF NOT EXISTS lti_placement (
+            id BIGINT NOT NULL AUTO_INCREMENT,
+            account_id INTEGER NOT NULL,
+            course_id INTEGER NOT NULL,
```

Comment:
course_id shall be a FK to course table

Reply:
Is account_id really needed here?

Reply:
This is some of the disconnect between the scripts and having different developers work on different parts with no overall framework.

MiVideo also has mivideo_media_courses that doesn't link back to the courses table

The course table is only populated by the inventory script. We'd have to refactor to allow any of these scripts to add courses to the courses table. The problem is the inventory script when it runs drops all the courses so that would be problematic. This sounds like a much bigger issue.

account_id is here because it comes from the api. I could remove it. All of these could be removed if we could get a FK link to courses.

Reply:
I'm creating a new issue to deal with this

Reply:
I created an issue #162 to resolve this